.github/RELEASE/subs2srs.conf

```diff
@@ -275,6 +287,15 @@ secondary_sub_area=0.15
 # Default binding to cycle this value: Ctrl+v.
 secondary_sub_visibility=auto
 
+# Perform loudness normalization.
+# Parameter explanation can be found e.g. at:
+# https://auphonic.com/blog/2013/01/07/loudness-targets-mobile-audio-podcasts-radio-tv/
+# https://auphonic.com/blog/2019/08/19/dynamic-range-processing/
+loudnorm=yes
+loudnorm_target=-16
+loudnorm_range=11
+loudnorm_peak=-1.5
```

Comment:
With the new config parameters, the users will have to update their config files. For example, I have previously set loudnorm parameters using `ffmpeg_audio_args`, and now I get audio files with different loudness.

Reply:
Disabled two-pass `loudnorm` by default and brought back the custom filter. Added scary warnings that filter must be removed before enabling two-pass `loudnorm`.

Loudness will be the same if target, range and peak are set to the same parameters that previously were in `ffmpeg_audio_args`.